tests/integration/init/schemas/schemas_test_data_setup.py

```diff
@@ -128,17 +115,28 @@ def _create_config_file(self, profile, region):
             file.write(config_file_content)
         return custom_config
 
-    def _create_cred_file(self, profile, access_key, secret_key):
-        cred_file_content = "[default]\naws_access_key_id = {1}\naws_secret_access_key = {2}\n"
+    def _create_cred_file(self, profile, access_key, secret_key, session_token=None):
+        cred_file_content = self._create_cred_profile("default", access_key, secret_key, session_token)
```

Comment:
Doesn't this need to be `[default]`?

Reply:
O... this is moved down to the helper method below.

Reply:
That is taken care of in the method line 133. It is `[{profile_name}]`